### ISSUE TITLE
Randomly resprings

### ISSUE THREAD

**Issue by Aem2512** [MTACS/FingerLock#8]
**Describe the bug**
-Device resprings sometimes when tapping button again from notification center, after unlocking device. Not always but sometimes this happens. (Not while in locked state) 

**Expected behavior**
-I expected to see phone returns to homescreen again like homebar/home button does. 

- Jailbreak: [Palera1n]
 - iOS Version: [15.4.1]
 - Package manager: [Sileo]
 - Tweak Version [3.0]

**Comment by MTACS** [comment]
I assume you have this option disabled? ![image](https://user-images.githubusercontent.com/13209789/213801396-5d63ff40-9fb6-496f-98e5-aaaa5d79a283.jpeg)

**Comment by Aem2512** [comment]
No, it was already turned on but still randomly respringed.  And i think this happens after i selected custom theme that i created. It not happens on stock finger icon. 

-AEM, (Emre)
